lib/components/util/toasts.tsx

```diff
@@ -31,14 +31,15 @@ export function toastSuccess(
 /**
  * Helper that will display a toast notification when a place is saved.
  */
-export function toastOnPlaceSaved(
+export function toastOnPlaceChanged(
   place: UserSavedLocation,
-  intl: IntlShape
+  intl: IntlShape,
+  change: 'Remembered' | 'Deleted'
 ): void {
   toastSuccess(
     getPlaceMainText(place, intl),
     intl.formatMessage({
-      id: 'actions.user.placeRemembered'
+      id: `actions.user.place${change}`
```

Comment:
I remember there was discussion a while ago about avoiding dynamic i18n string generation unless we really couldn't avoid it. Maybe a compromise could be 
```suggestion
      id: `actions.user.place.${change}`
```

Reply:
Yes, you're right I don't know why I always forget this! I added a less invasive conditional, lmk if that's still too much